src/repr/src/adt/datetime.rs

```diff
@@ -1362,9 +1376,12 @@ pub(crate) fn tokenize_time_str(value: &str) -> Result<VecDeque<TimeStrToken>, S
     let mut num_buf = String::with_capacity(4);
     let mut char_buf = String::with_capacity(7);
     fn parse_num(n: &str, idx: usize) -> Result<TimeStrToken, String> {
-        Ok(TimeStrToken::Num(n.parse().map_err(|e| {
-            format!("Unable to parse value as a number at index {}: {}", idx, e)
-        })?))
+        Ok(TimeStrToken::Num(
+            n.parse().map_err(|e| {
+                format!("Unable to parse value as a number at index {}: {}", idx, e)
```

Comment:
To make this match our other error messages could you include the value that failed to parse in the message:

```suggestion
                format!("Unable to parse value {} as a number at index {}: {}", n, idx, e)
```

Reply:
Fixed, thanks!